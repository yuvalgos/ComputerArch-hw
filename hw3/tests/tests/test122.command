./dflow_calc tests/test122.dat tests/test122.in p85 d128 p191 p192 p8 p184 d58 p59 p115 d39 p99 p16 p106 p101 p69 p121 p72 p12 p185 p48 p156 p87 p100 p187 p82 d72 d153 p93 d43 d100 d105 d73 p129 d169 d138 p77 d37 p72 p145 d143 p31 p137 d170 d194 d149 d93 p23 d2 d132 d167 p169 d16 p180 p47 p42 p107 p198 d96 d101 d123 d132 d20 d94 p31 p45 d190 p122 d118 d18 d95 d15 d71 d92 d50 p92 p48 p49 p123 p78 p47 d174 p168 p80 p32 p69 p80 d129 d97 p191 p194 p30 p48 p16 d76 p134 d71 p111 d97 d55 p36 d29 d109 p34 p97 p166 d41 p30 p8 p132 p92 d60 p176 p95 p99 d75 p197 p171 p12