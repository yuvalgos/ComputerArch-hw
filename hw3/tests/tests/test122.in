8 23 14 0
3 20 13 26
3 10 13 14
6 16 16 31
8 21 11 8
3 28 15 27
0 24 17 13
6 24 3 15
0 9 5 11
2 26 14 22
5 27 25 21
8 7 2 6
0 21 25 14
0 4 1 9
4 17 18 4
4 26 5 13
7 3 16 21
7 17 18 12
4 20 16 24
3 30 12 30
8 15 22 30
1 0 13 23
2 26 3 6
1 8 26 28
3 11 14 3
8 26 17 22
0 17 8 21
6 17 1 0
2 20 14 10
8 26 6 30
0 6 9 4
4 0 28 31
6 17 25 10
7 11 7 4
0 16 5 16
3 16 23 22
2 24 23 14
2 10 31 27
5 1 27 28
2 27 11 1
2 9 18 27
0 16 17 2
4 19 15 3
1 11 14 9
0 27 0 18
4 12 27 7
4 26 7 17
2 28 4 5
6 7 21 18
3 4 1 21
2 17 15 13
7 28 13 27
2 4 2 26
4 5 3 5
6 27 13 4
1 3 5 16
4 8 22 29
0 4 29 30
6 31 19 15
5 2 18 1
1 10 31 18
6 13 5 12
1 30 22 28
3 21 17 1
2 15 3 0
6 12 22 30
5 11 12 28
7 26 16 16
4 5 13 26
3 22 17 15
8 0 3 11
1 3 7 24
4 25 17 8
0 12 6 30
3 9 8 30
2 18 8 11
2 23 17 17
2 5 30 2
5 18 12 9
2 21 12 16
4 6 11 21
6 18 25 21
8 17 5 26
6 15 22 1
2 18 0 19
5 29 27 12
4 5 31 22
4 19 18 18
6 1 6 0
6 18 11 22
8 6 25 4
7 24 21 8
4 21 2 23
1 8 14 24
7 8 25 25
2 9 28 2
2 11 11 12
4 16 31 9
5 4 31 0
5 22 23 21
3 7 13 30
4 3 27 20
3 23 16 30
7 7 20 16
6 3 30 11
3 10 14 21
4 24 6 7
2 18 5 1
3 30 24 26
0 28 6 1
2 13 13 27
0 7 14 24
0 16 8 12
7 28 12 2
7 8 5 14
1 10 23 17
8 3 14 24
2 7 21 12
1 14 3 1
1 11 12 11
6 24 3 22
8 14 13 2
0 0 22 3
3 0 21 2
2 21 11 4
8 21 6 16
7 6 18 12
0 10 14 25
8 7 22 24
3 15 15 3
1 11 28 18
6 16 16 3
6 1 16 13
0 23 1 26
5 8 26 18
7 21 21 24
7 23 29 13
7 24 31 0
2 18 17 18
8 2 4 1
4 1 27 8
1 3 28 5
7 10 10 12
7 28 21 30
1 9 21 26
2 20 30 12
8 17 2 6
6 9 31 15
7 11 4 10
7 21 22 19
6 23 2 2
6 26 29 6
8 13 1 31
7 20 23 11
1 8 8 18
7 31 25 10
7 21 24 13
3 3 0 17
2 10 30 3
2 31 28 4
8 12 1 21
5 14 21 28
5 11 17 25
8 0 13 7
4 27 0 9
7 28 21 5
1 1 0 15
6 24 31 8
2 4 27 14
1 7 12 13
2 19 22 27
5 17 16 11
8 1 29 25
8 6 25 7
6 27 7 13
7 11 28 0
1 18 1 7
6 7 17 10
7 7 31 27
1 3 20 23
0 9 6 16
5 16 4 19
7 16 20 13
8 28 0 0
2 20 0 14
7 23 14 6
5 24 21 0
8 27 15 3
7 24 4 17
2 7 16 23
1 31 13 23
2 31 27 2
0 22 2 8
1 8 8 6
0 1 10 23
6 2 18 22
6 8 21 11
7 19 16 1
6 29 31 22
5 30 17 14
