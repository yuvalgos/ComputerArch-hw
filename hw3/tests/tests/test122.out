Reading the opcodes latency info from tests/test122.dat ... Got latency for 9 opcodes
Reading the program file tests/test122.in ... Found 200 instructions
getProgDepth()==272
getDepDepth(85)==57
getInstDeps(128)=={99,120}
getDepDepth(191)==250
getDepDepth(192)==188
getDepDepth(8)==0
getDepDepth(184)==199
getInstDeps(58)=={42,20}
getDepDepth(59)==49
getDepDepth(115)==90
getInstDeps(39)=={33,38}
getDepDepth(99)==73
getDepDepth(16)==17
getDepDepth(106)==97
getDepDepth(101)==57
getDepDepth(69)==55
getDepDepth(121)==69
getDepDepth(72)==35
getDepDepth(12)==0
getDepDepth(185)==232
getDepDepth(48)==2
getDepDepth(156)==164
getDepDepth(87)==100
getDepDepth(100)==64
getDepDepth(187)==227
getDepDepth(82)==67
getInstDeps(72)=={50,56}
getInstDeps(153)=={150,148}
getDepDepth(93)==68
getInstDeps(43)=={-1,40}
getInstDeps(100)=={61,62}
getInstDeps(105)=={-1,92}
getInstDeps(73)=={30,62}
getDepDepth(129)==131
getInstDeps(169)=={160,152}
getInstDeps(138)=={82,107}
getDepDepth(77)==62
getInstDeps(37)=={-1,10}
getDepDepth(72)==35
getDepDepth(145)==115
getInstDeps(143)=={135,108}
getDepDepth(31)==7
getDepDepth(137)==80
getInstDeps(170)=={99,164}
getInstDeps(194)=={158,185}
getInstDeps(149)=={99,87}
getInstDeps(93)=={-1,91}
getDepDepth(23)==41
getInstDeps(2)=={-1,-1}
getInstDeps(132)=={131,110}
getInstDeps(167)=={159,154}
getDepDepth(169)==206
getInstDeps(16)=={3,12}
getDepDepth(180)==232
getDepDepth(47)==4
getDepDepth(42)==36
getDepDepth(107)==112
getDepDepth(198)==255
getInstDeps(96)=={66,73}
getInstDeps(101)=={54,28}
getInstDeps(123)=={92,59}
getInstDeps(132)=={131,110}
getInstDeps(20)=={-1,19}
getInstDeps(94)=={72,72}
getDepDepth(31)==7
getDepDepth(45)==23
getInstDeps(190)=={152,185}
getDepDepth(122)==131
getInstDeps(118)=={116,88}
getInstDeps(18)=={3,7}
getInstDeps(95)=={51,59}
getInstDeps(15)=={-1,-1}
getInstDeps(71)=={48,36}
getInstDeps(92)=={59,76}
getInstDeps(50)=={20,-1}
getDepDepth(92)==62
getDepDepth(48)==2
getDepDepth(49)==49
getDepDepth(123)==73
getDepDepth(78)==49
getDepDepth(47)==4
getInstDeps(174)=={169,152}
getDepDepth(168)==205
getDepDepth(80)==72
getDepDepth(32)==7
getDepDepth(69)==55
getDepDepth(80)==72
getInstDeps(129)=={83,116}
getInstDeps(97)=={58,95}
getDepDepth(191)==250
getDepDepth(194)==251
getDepDepth(30)==4
getDepDepth(48)==2
getDepDepth(16)==17
getInstDeps(76)=={50,50}
getDepDepth(134)==117
getInstDeps(71)=={48,36}
getDepDepth(111)==108
getInstDeps(97)=={58,95}
getInstDeps(55)=={53,41}
getDepDepth(36)==23
getInstDeps(29)=={-1,19}
getInstDeps(109)=={90,88}
getDepDepth(34)==17
getDepDepth(97)==67
getDepDepth(166)==194
getInstDeps(41)=={32,-1}
getDepDepth(30)==4
getDepDepth(8)==0
getDepDepth(132)==148
getDepDepth(92)==62
getInstDeps(60)=={58,-1}
getDepDepth(176)==209
getDepDepth(95)==62
getDepDepth(99)==73
getInstDeps(75)=={56,66}
getDepDepth(197)==253
getDepDepth(171)==224
getDepDepth(12)==0
